_data/locales/pt.yml

```
@@ -1,5 +1,5 @@
 pt:
-  locale_name: Português
+  locale_name: Português (europeu)
```

```suggestion
  locale_name: Português
```

is more typical, no?